example/bwmf/bwmf_task.go

```diff
@@ -393,7 +407,7 @@ func (t *bwmfTask) GetDShard(ctx context.Context, request *pb.Request) (*pb.Resp
 	return resp, nil
 }
 
-func (t *bwmfTask) MetaReady(ctx context.Context, fromID uint64, linkType, meta string) {
+func (t *bwmfTask) MetaReady(ctx context.Context, fromID uint64, meta string) {
```

Comment:
yes, it is app specific thing, but should we support that through interface?

On Thu, May 21, 2015 at 7:33 AM, Hongchao Deng <notifications@github.com>
wrote:

> In example/bwmf/bwmf_task.go
> <https://github.com/taskgraph/taskgraph/pull/121#discussion_r30760607>:
>
> > @@ -393,7 +407,7 @@ func (t *bwmfTask) GetDShard(ctx context.Context, request *pb.Request) (*pb.Resp
> >  	return resp, nil
> >  }
> >
> > -func (t *bwmfTask) MetaReady(ctx context.Context, fromID uint64, linkType, meta string) {
> > +func (t *bwmfTask) MetaReady(ctx context.Context, fromID uint64, meta string) {
>
> Linktype should be app logic specific thing.
>
> Take Parents/Children for example, in application, if A send data to
> "parent" B, B knows it's from "children" A.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/taskgraph/taskgraph/pull/121/files#r30760607>.
>